wavesrv/db/migrations/000029_canonicalport.up.sql

```diff
@@ -0,0 +1,4 @@
+UPDATE remote
+SET remotecanonicalname = case when json_extract(sshopts, '$.issudo') then 'sudo@' else '' end ||
```

Comment:
I think you should just assume that remotecanonicalname is correct in the DB.  Because you're already checking that sshport is not 22 (and implicitly not null).  So then you can just do:
```
SET remotecanonicalname = remotecanonicalname || ':' || json_extract(sshopts, '$.sshport')
```